sdk/mobilenetwork/arm-mobilenetwork/src/models/mappers.ts

```diff
@@ -3292,7 +3412,7 @@ export const PacketCapture: coreClient.CompositeMapper = {
         defaultValue: 67108864,
         constraints: {
           InclusiveMaximum: 4294967295,
-          InclusiveMinimum: 0
+          InclusiveMinimum: 1000
```

Comment:
feel like this is a breaking change. because what used to work like under 1000 but not now.

Reply:
this change is approved in api level https://github.com/Azure/azure-rest-api-specs/pull/26133#issuecomment-1752660925

Reply:
we should bump major version here.

Reply:
We never released PacketCapture in the old API, so we are safe to release without a major bump.

Reply:
the last release version is [2023-06-01](https://github.com/Azure/azure-rest-api-specs/tree/main/specification/mobilenetwork/resource-manager#tag-package-2023-06) which contains PacketCapture, so we should bump a major version 
last release request: https://github.com/Azure/sdk-release-request/issues/4278